mne/viz/raw.py

```diff
@@ -235,10 +235,21 @@ def plot_raw(raw, events=None, duration=10.0, start=0.0, n_channels=20,
         inds += [pick_types(info, meg=t, ref_meg=False, exclude=[])]
         types += [t] * len(inds[-1])
     pick_kwargs = dict(meg=False, ref_meg=False, exclude=[])
-    for t in ['eeg', 'eog', 'ecg', 'emg', 'ref_meg', 'stim', 'resp',
+    for t in ['eeg', 'seeg', 'eog', 'ecg', 'emg', 'ref_meg', 'stim', 'resp',
               'misc', 'chpi', 'syst', 'ias', 'exci']:
         pick_kwargs[t] = True
         inds += [pick_types(raw.info, **pick_kwargs)]
+        if t == 'seeg' and len(inds[-1]) > 0:
+            # XXX hack to work around fiff mess
+            new_picks = [ind for ind in inds[-1] if
+                         not 'HPI' in raw.ch_names[ind]]
```

Comment:
(but please do `not in` instead of `not x in`, it's a PEP8 thing and actually invokes a different builtin function for some classes so it's a good habit to have)